packages/typespec-python/src/http.ts

```diff
@@ -69,7 +69,7 @@ function emitInitialLroHttpMethod(
     operationGroupName: string,
 ): Record<string, any> {
     return {
-        ...emitHttpOperation(context, rootClient, operationGroupName, method.operation),
+        ...emitHttpOperation(context, rootClient, operationGroupName, method.operation, method),
```

Comment:
Python SDK always set response type of initial operation as Binary type so just keep it same as `begin_xxx` API.